### DISCUSSION TITLE
A 32x developpement

### DISCUSSION THREAD

**Discussion by maiden312** [flybywiresim/aircraft#8595] [General]
bonjour j ai un probléme avec la version développement de l A 32x j ai les écrans du du tableau de bord  ou il y a l altitude et cap qui sont décales et tout petit ont ne les voit presque pas et le bouton contrainte qui ne fonctionne pas quelqu un pourrait il m aider merci d avance

**Comment by frenchairbuspilot** [comment]
HI maiden,
La prochaine fois essaie d'écrire correctement et de traduire ton message en anglais je te prie, tu auras plus de support ainsi.
Ton problème vient surement d'une texture/livrée qui n'est plus supporté sur le modèle actuel de l'avion.
Essai de trouver une livrée plus récente.

jp

**Comment by maiden312** [comment]
Merci

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
